src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2364,4 +2366,8 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="Run_ConflictingKeywordInfo_Link.Content" xml:space="preserve">
     <value>Learn more about conflicting activation commands</value>
   </data>
-</root>
+  <data name="MeasureTool_ContinuousCapture_Information.Title" xml:space="preserve">
+    <value>This mode will consume more resources when in use.  In addition, the UI will be slightly offset from the pointer position.</value>
```

Comment:
```suggestion
    <value>The continues capture mode will consume more resources when in use.  In addition, the UI will be slightly offset from the pointer position.</value>
```

That the user understand for which setting the warning is

Reply:
@htcfreek : "continuous"

Reply:
The warning does appear right next to the setting, by the way.

Reply:
Added a commit to add this suggestion. Thank you!